src/java/com/github/prasanthj/hll/HLLSparseRegister.java

```diff
@@ -203,6 +204,18 @@ public Int2ByteSortedMap getMergedSparseMap() {
     return sparseMap;
   }
 
+  // this is effectively the same as the dense register impl.
+  public void extractLowBitsTo(HLLRegister dest) {
+    for (Entry<Integer, Byte> entry : sparseMap.entrySet()) {
+      int idx = entry.getKey();
+      byte lr = entry.getValue(); // this can be a max of 65, never > 127
+      if (lr > 0) {
+        // should be a no-op for sparse
```

Comment:
why is this a no-op? 
this is essentially the same as dense except that it does not iterate from 0 to 2^p.

Reply:
The sparse is missing all the unset bytes